src/Window.vala

```diff
@@ -88,6 +90,8 @@ public class Akira.Window : Gtk.ApplicationWindow {
 	public bool before_destroy () {
 		update_status ();
 
+		save_and_close_current_file ();
```

Comment:
Can we store the uri of the latest opened file here? If you're planning to do that later, just ignore my comment here.

Reply:
The window actually already stores a local reference to the file it has opened called AkiraFile. :)

`public FileFormat.AkiraFile? akira_file = null;`